spec/aggregate_root_spec.rb

```diff
@@ -3,16 +3,18 @@
 class Order
   include RailsEventStore::AggregateRoot
 
+  attr_accessor :id, :status
```

Comment:
Right I agree with you. There is no clear way yet to assert event creation. Maybe the gem should provide a custom rspec validator? If we go that path though, we need to handle more than rspec...